examples/actors/mutate_remote_state.py

```diff
@@ -0,0 +1,51 @@
+from itertools import cycle
+from pprint import pformat
+
+import trio
+import tractor
+
+_snd_chan, _recv_chan = trio.open_memory_channel(100)
+_actor_state = {'some_state_stuff': None}
```

Comment:
This is a module level variable meaning it maintains it's state for the entire process lifetime.
We could have just as easily made some function that "sleep forever" and wakes up incrementally to report it's state if need be.